README.md

```diff
@@ -1 +1,5 @@
 # opencensus-go-resource
+
```

Comment:
Can you add a line with the import path? So the users know where to import the package from?